docs/api/declarative-routers/unstable_HistoryRouter.md

```diff
@@ -1,36 +0,0 @@
----
-title: HistoryRouter
```

Comment:
This will now be auto-generated as just `HistoryRouter.md` so the URL stays stable even if we ever stabilize the API (which we won't in this case, but now it's consistent with how we handle `usePrompt` and others)